services/payment-service/main.py

```
@@ -2,16 +2,25 @@
 from fastapi.middleware.cors import CORSMiddleware
 from prometheus_client import make_asgi_app, Counter, Histogram, Gauge
 import time
+from prometheus_client import make_asgi_app, Counter, Histogram, Gauge
```

Ruff (0.12.2)

Redefinition of unused `make_asgi_app` from line 3

Remove definition: `make_asgi_app`

(F811)